guides/source/action_text_overview.md

```diff
@@ -163,24 +159,62 @@ To customize the HTML rendered for embedded images and other attachments (known
       <span class="attachment__name"><%= blob.filename %></span>
       <span class="attachment__size"><%= number_to_human_size blob.byte_size %></span>
     <% end %>
-  </figcaption>
-</figure>
+  </figcapt
 ```
 
-### Rendering attachments
+## Attachments
+
+Currently Action Text supports attachments that are uploaded through Active Storage as well as attachments that are linked to a Signed GlobalId.
+
+### Active Storage
+
+When uploading an image within your rich text editor, it uses Action Text which in turn uses Active Storage. However, Active Storage has [some dependencies](active_storage_overview.html#requirements) which are not provided by Rails. To use the built-in previewers, you must install these libraries.
+
+Some, but not all of these libraries are required and they are dependent on the kind of uploads you are expecting within the editor. A common error that users encounter when working with Action Text and Active Storage is that images do not render correctly in the editor. This is usually due to the `libvips` dependency not being installed.
+
+
+### Signed GlobalId
+
+In addition to attachments uploaded through Active Storage, Action Text can also embed anything that can be resolved by a Signed GlobalID.
```

Comment:
It looks like we lost the linking to the globalid repo, it might be nice to keep it around for a reference there.

```suggestion
In addition to attachments uploaded through Active Storage, Action Text can also embed anything that can be resolved by a [Signed GlobalID](https://github.com/rails/globalid#signed-global-ids).
```